openslides/core/views.py

```diff
@@ -24,6 +25,14 @@
 from .serializers import CustomSlideSerializer, TagSerializer
 
 
+class AngularView(utils_views.View):
```

Comment:
Why not use a simple TemplateView? It means less code and plugins may easily override the default index template.